Mail/Views/Thread List/ThreadListViewModel+Observation.swift

```diff
@@ -41,7 +41,7 @@ extension ThreadListViewModel {
 
     // MARK: - Observe global changes
 
-    func observeChanges(animateInitialThreadChanges: Bool = false) {
+    func observeChanges() {
```

Comment:
Now we always perform an animation. I seems like doesn't have any side effect even at initial launch.